JASMIN/catalog-JASMIN.yaml

```diff
@@ -42,3 +42,46 @@ sources:
         max: 2015
         min: 1958
         type: int
+
+  ORCHESTRA_ZPS_CORE2NYF:
+    args:
+      urlpath: /bodc/BAS210039/CORE2NYF-ORCH0083-LIM3/{{ year }}/{{ time_freq }}/{{ grid }}/*.nc
+      xarray_kwargs:
+        compat: override
+        coords: minimal
+        data_vars: minimal
+        decode_timedelta: false
+        parallel: parallel
+    description: TODO
+    driver: netcdf
+    metadata:
+      catalog_entry_author: Adam Blaker
+    parameters:
+      grid:
+        allowed:
+        - I
+        - T
+        - U
+        - V
+        default: T
+        description: grid
+        type: str
+      parallel:
+        default: true
+        description: If True, the open and preprocess steps of this function will
+          be performed in parallel using `dask.delayed`.
+        type: bool
```

Comment:
I like changing the default of parallel to `True`, so if dask is installed and there are multiple workers it is much faster. Can be confusing though, so we can remove it and the user will have to explicitly pass `parallel` using `xarray_kwargs`.

Reply:
I agree. Default parallel is good.

Reply:
Need to add " - W " grid (annoyingly "I" is only available at daily, and "W" is available at d05 and m01)